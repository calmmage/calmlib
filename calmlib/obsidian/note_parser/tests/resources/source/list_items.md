- regular list item
- another item
    - nested item
- [ ] checkbox item
- [x] checked item
