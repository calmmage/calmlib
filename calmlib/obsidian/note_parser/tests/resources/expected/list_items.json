{
  "items": [
    {
      "type": "list_item",
      "text": "regular list item"
    },
    {
      "type": "list_item",
      "text": "another item",
      "children": [
        {
          "type": "list_item",
          "text": "nested item"
        }
      ]
    },
    {
      "type": "list_item",
      "text": "checkbox item",
      "checked": false
    },
    {
      "type": "list_item",
      "text": "checked item",
      "checked": true
    }
  ]
}